pkg/standup/standup.go

```diff
@@ -58,6 +59,12 @@ func (c *Client) Standup(cluster *application.Cluster) (*application.Cluster, er
 	applyCtx, cancelApplyCtx := context.WithTimeout(ctx, 20*time.Minute)
 	defer cancelApplyCtx()
 
+	skipsDefaultApps, err := cluster.UsesUnifiedClusterApp()
+	Expect(err).NotTo(HaveOccurred())
+	if skipsDefaultApps {
+		logger.Log("Deploying only unified %s app (with default apps) and skipping %s app.", cluster.ClusterApp.AppName, cluster.DefaultAppsApp.AppName)
+	}
+
```

Comment:
I think we can leave this out to be honest. It's just extra code we'll need to remove when all providers are complete.

Reply:
Sure, will remove it.

Reply:
Removed.